content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,41 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind Persistent Volume Claims to matching Persistent Volumes](#binding) in the
+cluster. However, if you want a Persistent Volume Claim to bind to a specific Persistent Volume, you need to pre-bind them.
+
+You can specify the Persistent Volume in your Persistent Volume Claim, then the Persistent Volume Claim will be able to bind to this specific Persistent Volume. If the Persistent Volume exists and is not reserved for other Persistent Volume Claims (has Persistent Volume Claim specified in `claimRef` field), the Persistent Volume and Persistent Volume Claim will be bound regardless of whether the Persistent Volume satisfies some matching criteria, e.g. node affinity. Storage class, access modes, and requested storage size are still checked.
```

Comment:
The last part of the sentence is a little confusing. It says it will be bound regardless of matching criteria, but then we have a list of criteria that is still checked.

Reply:
Does this help:

By setting a PV in a Persistent Volume Claim, you declare a binding between a specific Persistent Volume and a PVC.
If the Persistent Volume exists and has not reserved PersistentVolumeClaims through its `claimRef` field, then the Persistent Volume and Persistent Volume Claim will be bound regardless of whether the Persistent Volume satisfies some matching criteria such as node affinity. Storage class, access modes, and requested storage size are still checked.

Can the binding fail depending on the outcome of the items in the last sentence?

Reply:
Ah sorry. I missed the period after affinity. This makes a lot more sense now.

Reply:
```suggestion
By specifying a PersistentVolume in a PersistentVolumeClaim, you declare a binding between that specific PV and PVC.
If the PersistentVolume exists and has not reserved PersistentVolumeClaims through its `claimRef` field, then the PersistentVolume and PersistentVolumeClaim will be bound.

The binding happens regardless of some volume matching criteria, including node affinity.
The control plane still checks that [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/), access modes, and requested storage size are valid.
```
?